indexwarcsjob.py

```diff
@@ -106,6 +106,8 @@ def _conv_warc_to_cdx_path(self, warc_path):
         # set cdx path
         cdx_path = warc_path.replace('crawl-data', 'cc-index/cdx')
         cdx_path = cdx_path.replace('.warc.gz', '.cdx.gz')
+        cdx_path = cdx_path.replace('.wet.gz', '.wet.cdx.gz')
```

Comment:
Shouldn't the "complete" three-part suffix  `.warc.wet.gz` (or `.warc.wat.gz`) be replaced?
- `foobar.warc.wet.gz` -> `foobar.wet.cdx.gz`
- `foobar.warc.gz` -> `foobar.cdx.gz` 

The CDX filename from the WARC file does not include a ".warc" - it's no WARC file. WET files are WARC files which result from a specific transformation.

Reply:
I don't think there is a perfect way to handle - but I submitted a commit to capture the full name this time. I sill preserve the wet/wat in the cdx filename, since there wouldn't otherwise be a way to differentiate between.

Reply:
> Of course, if there are WET files without the .warc in the file suffix, we may map these suffixes, too.

@sebastian-nagel  - are you saying there are both xxx.warc.wet.gz and xxx.wet.gz wet files? if so, I will add both patterns.

Reply:
I am doing a full file listing of commoncrawl/crawl-data to confirm the filename patterns which exist. It will probably take a few hours. @wumpus or @sebastian-nagel  - if you know the answer to my above question about filename suffixes, please comment and I will stop the listing.

Reply:
OK, I do not see any other patterns, code should be good as-is.

Reply:
Sorry, I missed this question. Yes. At Common Crawl, all WET files are named ".warc.wet.gz", but other organizations may use a different naming scheme.

Reply:
The main other archive that has WAT and WET is the EOT (End of Term) Archive, and they also use the same filenames as us, warc.wet.gz and warc.wat.gz